bitnami/argo-cd/templates/repo-server/deployment.yaml

```diff
@@ -137,6 +137,40 @@ spec:
                   key: {{ include "argocd.redis.secretPasswordKey" . }}
           {{- end }}
         {{- end }}
+        {{- if .Values.repoServer.configManagementPlugins.enabled }}
+        - name: copy-cmp-server-binary
+          image: {{ include "argocd.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - /bin/cp
+            - -n
+            - /opt/bitnami/argo-cd/bin/argocd
+            - /additional-binaries/argocd-cmp-server
+          {{- if .Values.repoServer.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.repoServer.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - mountPath: /additional-binaries
+              name: cmp-additional-binaries
+        {{- if .Values.repoServer.configManagementPlugins.additionalBinaries }}
+        - name: download-additional-binaries
+          image: {{ include "common.images.image" (dict "imageRoot" (dict "repository" "curlimages/curl" "tag" "latest") "global" .Values.global) }}
```

Comment:
Any container image should be maintained by Bitnami and configured in a standard way from `values.yaml`. You could use `bitnami/os-shell`. Look at the `volumePermissions.image` value to follow a similar pattern.

Do you have any concerns with that?

Reply:
No issues at all! I remember thinking about this when I added the init-container. I think I just forgot about it. Anyway, added it now!